pages/auction/index.js

```diff
@@ -15,9 +15,11 @@ const Auctions = () => {
       </Head>
       <NavBar personData={personData} />
       <Link href="/auction/new">
-        <a>
-          <h2 className={styles.auctionPageNavigation}>Create New Auction</h2>
-        </a>
+        <div className={styles.newAuctionBtnContainer}>
+          <button className={styles.newAuctionButton}>
```

Comment:
Better to go with `newAuctionBtn`?

Reply:
Yes swaraj, done with all the changes. Thank you!

Reply:
![image](https://user-images.githubusercontent.com/86421817/128734952-ee6dea61-315e-44b1-aa48-3527e1301f4a.png)
Before putting the button. 
![image](https://user-images.githubusercontent.com/86421817/128735595-2b1f1bf4-4b08-4107-80e9-
4ddee157d766.png)
After putting the button and applying css.